bin/sawtooth

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
-# Copyright 2016 Intel Corporation
+# Copyright 2017 Intel Corporation
```

Comment:
According to @TomBarnes, the correct way to update a copyright is "Copyright 2016, 2017 Intel Corporation" if the file was created in 2016 and updated in 2017.